src/main/java/net/minecraftforge/server/permission/IPermissionHandler.java

```diff
@@ -38,9 +39,9 @@
     Collection<String> getRegisteredNodes();
 
     /**
-     * Use {@link PermissionAPI#hasPermission(GameProfile, String, IContext)}
+     * Use {@link PermissionAPI#hasPermission(MinecraftServer, GameProfile, String, IContext)}
      */
-    boolean hasPermission(GameProfile profile, String node, @Nullable IContext context);
+    boolean hasPermission(@Nullable MinecraftServer server, GameProfile profile, String node, @Nullable IContext context);
```

Comment:
Why did you change the API here? I understand the old TODO called some helper method to get the current server. So it would be better to get the server internally instead of expecting everyone to send it in.

Reply:
I did not know that there was a replacement function for the static server instance.